studio/components/interfaces/Database/Hooks/HTTPRequestFields.tsx

```diff
@@ -185,9 +194,9 @@ const HTTPRequestFields = ({
                           Required if your edge function enforces JWT verification
                         </p>
                       </div>
-                    </Dropdown.Item>,
-                    <Dropdown.Separator key="separator" />,
-                    <Dropdown.Item
+                    </DropdownMenuItem_Shadcn_>
+                    <DropdownMenuSeparator_Shadcn_ key="separator" />
```

Comment:
`key` is probably no longer needed here due to this not being a list anymore. That being said, it's more of a nit than an actual issue!